README.md

```diff
@@ -34,3 +34,78 @@ Triggers, and any other element that can be used with `tektoncd/pipeline`).
   - Possibly driving API "feature"/changes
     Because we will write a lot of task, use them, … we should be able to find gap or
     enhancements in the API, and propose them as TEPs (with data).
+
+# Usage Examples
+
+
+This section explains how to use the tasks supported in this repository with the help of various tools like Tekton Resolvers as well as Pipelines as Code. 
+
+## Using Tekton Resolvers
```

Comment:
I think it would be even more interesting to show the example in a `Pipeline` (+`PipelineRun`) as this should be 99% of use cases for most tasks.